org.eclipse.lsp4xml/src/test/java/org/eclipse/lsp4xml/extensions/contentmodel/DTDDoctypeDiagnosticsTest.java

```diff
@@ -90,4 +94,25 @@ public void disableDTDValidationWhenNoElementDecl() throws Exception {
 		// declaration
 		XMLAssert.testDiagnosticsFor(xml);
 	}
+
+	@Test
+	public void doctypeNotAllowed() throws Exception {
+		String xml = "<?xml version=\"1.0\"?>\r\n" + //
+				"<!DOCTYPE student [\r\n" + // <-- error DOCTYPE is disallow
+				"  <!ELEMENT \r\n" + //
+				"]>\r\n" + //
+				"<student />";
+		testDiagnosticsDisallowDocTypeDecl(xml, d(1, 0, 3, 2, XMLSyntaxErrorCode.DoctypeNotAllowed));

```

Comment:
severity should be configurable

Reply:
Or not...

Reply:
Today none severity can be configured because there are so many error code.